templates/plugin/.taprc

```diff
@@ -0,0 +1,4 @@
+esm: false
```

Comment:
We can use the same config as it is here: https://github.com/fastify/fast-json-stringify/blob/master/.taprc

Why setting jobs to 1?

Reply:
I simply used the same `.taprc` of the [fastify-cli](https://github.com/fastify/fastify-cli/blob/master/.taprc).
But maybe it was not a good idea since the fastify-cli is a bit different from a simple plugin.
I'll take your suggestion.